perception/navigator_vision/nodes/object_classifier.py

```diff
@@ -61,14 +59,14 @@ def database_request(self, req):
 
 @util.cancellableInlineCallbacks
 def main():
-    nh = yield txros.NodeHandle.from_argv("object_classifier")
-    config = Config()
+    # nh = yield txros.NodeHandle.from_argv("object_classifier")
+    # config = Config()
     class_file = os.path.abspath(__file__)
     class_file = class_file.split("nodes")[0]
     class_file = class_file + "object_classification/train.p"
 
-    cl = depicklify(class_file)
-    oc = yield ObjectClassifier(nh, cl, config).init_()
+    # cl = depicklify(class_file)
+    # oc = yield ObjectClassifier(nh, cl, config).init_()
```

Comment:
This will break the program! Just replace ```oc = yield ObjectClassifier(nh, cl, config).init_()``` with ```yield ObjectClassifier(nh, cl, config).init_()```. There are a couple other places this happens.